fw-child/template/query/app-item.php

```diff
@@ -1,29 +1,73 @@
 <?php
 
+$related_content = get_field( 'related_content', $item['id'] );
+
 ?>
 
-<div class="card">
-	<?php
-	
-		if ( has_post_thumbnail ( $item['id'] ) ) {
-			
-	?>
+<div class="card no-resize has-links">
 	
-	<div class="bg-dark">
-		<div class="card-img item-thumb" style="background-image: url(<?php echo get_the_post_thumbnail_url( $item['id'], 'medium_large' ); ?>);"></div>
+	<div class="bg-gray-400 border-bottom border-gray-400">
+		<a class="hover-toggle" href="<?php echo $item['permalink']; ?>">
+			<div class="card-img item-thumb" <?php echo has_post_thumbnail( $item['id'] ) ? 'style="background-image: url(' . get_the_post_thumbnail_url( $item['id'], 'medium_large' ) . ');"' : '' ?>></div>
+		</a>
 	</div>
 	
-	<?php
-	
-		}
-
-	?>
-	
-	<div class="card-body">
-		<h5 class="card-title item-title"><a href="<?php echo $item['permalink']; ?>" class="text-secondary stretched-link"><?php echo $item['title']; ?></a></h5>
+	<div class="card-body d-flex flex-column">
+		<h5 class="card-title item-title">
+			<a href="<?php echo $item['permalink']; ?>" class="text-secondary"><?php echo $item['title']; ?></a>
+		</h5>
 		
-		<div class="">
+		<div>
 			<?php echo apply_filters ( 'the_content', fw_get_field ( 'app_brief', $item['id'] ) ); ?>
 		</div>
+
+		<p class="text-end pt-3 mt-auto mb-1">
+			<a class="text-primary fw-bold" href="<?php echo $item['permalink']; ?>" class="text-secondary"><?php _e( 'Launch App', 'cdc' ) ?></a><a class="text-primary fw-bold text-decoration-none" href="<?php echo $item['permalink']; ?>" class="text-secondary"> &rarr;</a>
```

Comment:
Both a tags are purposely on the same line to avoid a space with no link between the text and the arrow, provide continuous link. It's been coded in two link to avoid underlining the arrow.